development/eks-d.tpl

```diff
@@ -0,0 +1,143 @@
+apiVersion: kops.k8s.io/v1alpha2
+kind: Cluster
+metadata:
+  creationTimestamp: null
+  name: {{ .clusterName }}
+spec:
+  api:
+    dns: {}
+  authorization:
+    rbac: {}
+  channel: stable
+  cloudProvider: aws
+  configBase: {{ .configBase }}
+  containerRuntime: docker
+  etcdClusters:
+  - cpuRequest: 200m
+    etcdMembers:
+    - instanceGroup: master-us-west-2a
```

Comment:
Some of the 'master' terminology is hard-coded (ex: `masterPublicName` below), but for variable names, lets use 'leader' (unless you think of something better)

And can you template out the region everywhere? 
```suggestion
    - instanceGroup: leader-{{{{.awsRegion}}}}a
```

Reply:
`control-plane` is the term we'll migrate towards in kOps, if that helps.

Reply:
I agree there were a lot of variables I wanted to pull out of the template and update names but I also didn't want to change too much in this PR to keep functionality the same and later go through and add some more options.